packages/webdriver/protocol/webdriver.json

```diff
@@ -662,7 +662,7 @@
         "required": true
       }, {
         "name": "args",
-        "type": "(string|object|number|boolean)[]",
+        "type": "(string|object|number|boolean|null|undefined)[]",
```

Comment:
we do a `typeof` check and `typeof null === "object"` so I think we can remove `null` here.